_scss/_mobile.scss

```diff
@@ -117,16 +118,10 @@
     .header {
         height: 100px;
     }
-    .nav-secondary-tabs,
-    .nav-secondary {
-        background-color: $black;
```

Comment:
Looks good so far, only thing I see is that on mobile the top bar is blue in light mode, but when I rotate my mobile to landscape it turns black. 
Previously on mobile it always was black. I think this change here was for black -> blue.
I like the blue background better in light mode.
But we should also fix it for landscape mode further down line 234 ff.

Reply:
Right, what I noticed was that on mobile the navbar (top bar) was black so I was trying to make it consistent with non-mobile views. I will change this for landscape mode as well. Nice find!